readthedocs/projects/admin.py

```diff
@@ -110,9 +110,13 @@ class ProjectAdmin(GuardedModelAdmin):
     search_fields = ('slug', 'repo')
     inlines = [ProjectRelationshipInline, RedirectInline,
                VersionInline, DomainInline]
+    readonly_fields = ('feature_flags',)
     raw_id_fields = ('users', 'main_language_project')
     actions = ['send_owner_email', 'ban_owner']
 
+    def feature_flags(self, obj):
+        return ', '.join([f.feature_id for f in obj.features])
```

Comment:
Isn't nicer to use the feature_display?
https://github.com/rtfd/readthedocs.org/blob/b05c82acc69e492533da9b7e3e40fef2456af6de/readthedocs/projects/models.py#L1049

Reply:
Ok